ssl/s3_lib.c

```diff
@@ -3545,20 +3545,23 @@ long ssl3_ctx_callback_ctrl(SSL_CTX *ctx, int cmd, void (*fp) (void))
     return (1);
 }
 
+const SSL_CIPHER *ssl3_get_cipher_by_id(uint32_t id)
+{
+    SSL_CIPHER c;
+
+    c.id = id;
+    return OBJ_bsearch_ssl_cipher_id(&c, ssl3_ciphers, SSL3_NUM_CIPHERS);
```

Comment:
Depends whether you mean just rename ssl3_ciphers, or whether you mean rename all ssl3_* things. If the former then I'd prefer you wait until after this PR has gone in...but then go for it. If the latter, then I'd prefer after more 1.3 stuff has gone through...just to avoid unnecessary churn and distraction.